### PR TITLE
Ruff suggests using a generator instead of tuple of list comprehension.

### PR DESCRIPTION
**Pull request by nkoskelo**

### CODE REVIEW

File: pytato/reductions.py [hunk 1]
```diff
@@ -178,10 +178,10 @@ def _normalize_reduction_axes(
             raise ValueError(f"{axis} is out of bounds for array of dimension"
                     f" {len(shape)}.")
 
-    new_shape = tuple([axis_len
+    new_shape = (axis_len
         for i, axis_len in enumerate(shape)
-        if i not in reduction_axes])
-    return new_shape, reduction_axes
+        if i not in reduction_axes)
+    return tuple(new_shape), reduction_axes
```
Comment (kaushikcfd) [inline pytato/reductions.py:184]:
```suggestion
    return new_shape, reduction_axes
```

File: pytato/reductions.py [hunk 2]
```diff
@@ -178,10 +178,10 @@ def _normalize_reduction_axes(
             raise ValueError(f"{axis} is out of bounds for array of dimension"
                     f" {len(shape)}.")
 
-    new_shape = tuple([axis_len
+    new_shape = (axis_len
```
Comment (kaushikcfd) [inline pytato/reductions.py:181]:
```suggestion
    new_shape = tuple(axis_len
```

**Review by kaushikcfd — APPROVED** [review]
Minor readability suggestion. LGTM!

**Comment by inducer** [comment]
Looks like the same changes were introduced elsewhere. Closing here.

**Pull request closed without merging**